packages/react-table/src/components/Table/Table.tsx

```diff
@@ -243,7 +246,6 @@ export class Table extends React.Component<TableProps, {}> {
           role={role}
           variant={variant}
           borders={borders}
-          className={className}
```

Comment:
should this be removed?

Reply:
I had changed the props a handful of times while working on this, so in the end while cleaning stuff up, I guess I realized that `className` was being pulled out of `props` unnecessarily on line 161. I can put it back, but it's now passed when I spread props to the Provider on line 230.